storage/innobase/xtrabackup/src/file_utils.cc

```diff
@@ -376,23 +376,5 @@ File open_fifo_for_read_with_timeout(const char *path, uint timeout) {
     return -1;
   }
 
-  /* File was open, lets check its open on the other side */
```

Comment:
where is this handled now? in my_open()?

Reply:
Fixed. Added support for MacOS Kqueue (mac's replacement of epoll)